xds/src/main/java/io/grpc/xds/EnvoyServerProtoData.java

```diff
@@ -201,259 +167,91 @@ public String toString() {
    * Corresponds to Envoy proto message
    * {@link io.envoyproxy.envoy.api.v2.listener.FilterChainMatch}.
```

Comment:
I think it will be a good idea to also replace v2 refs with the v3 ones in this PR. In this case `io.envoyproxy.envoy.config.listener.v3.FilterChainMatch`